lib/cc/analyzer/filesystem.rb

```diff
@@ -47,6 +37,12 @@ def files_matching(globs)
 
       private
 
+      def file_paths
+        @file_paths ||= Dir.chdir(@root) do
+          `find . -type f`.strip.split("\n")
```

Comment:
Did you find this to be much faster? @noahd1 and I found that it's the actual file access that's slow, not the ruby. In other words when we ran this `find` command in boot2docker on the large test repo, it was just as slow as the `Dir.glob` approach. We weren't rigorous though so I can't cite numbers. It could be faster, it just didn't feel it.

I'm fine making this change, just FYI it might not be the silver bullet.